ibis/backends/sql/compilers/risingwave.py

```diff
@@ -49,6 +44,12 @@ def visit_Correlation(self, op, *, left, right, how, where):
             op, left=left, right=right, how=how, where=where
         )
 
+    def visit_First(self, op, *, arg, where):
+        return self.agg.first_value(arg, where=where)
```

Comment:
This is implemented this way as a test case for the definition order, but it turns out to be needed in #9729.